samples/fastlane/Fastfile

```diff
@@ -57,5 +55,39 @@ platform :android do
       groups: test_groups,
       release_notes: build_notes
     )
-  end 
-end 
+end
+
+platform :android do 
+  lane :build do |values|
+    # Get the name of the Gradle module for the sample app. We need this later when building it. 
+    # The name of the gradle module is the name of the directory where the sample app source code is hosted in `samples/`. 
+    path_to_root_directory_sample_app = File.expand_path('../', Dir.pwd) # We are currently in `samples/N/fastlane/` and we need the directory name `samples/N``
+    name_of_sample_app_module = File.basename(path_to_root_directory_sample_app) # just get the name of the directory I am in now. `samples/N`
+
+    new_app_version = get_new_app_version()
+    test_groups = get_build_test_groups()
+    app_package_name = CredentialsManager::AppfileConfig.try_fetch_value(:package_name) # get package_name from Appfile
+
+    build_sample_app(
+        sample_app_name: name_of_sample_app_module,
+        app_version: new_app_version,
+        firebase_test_groups: test_groups,
+        app_package_name: app_package_name
+    )
+  end
+
+  lane :build_sample_app_for_sdk_release do |values|
+     # Apps use `build.gradle` or `build.kts`. This wildcard lookup is used to get the absolute path to one of these files present
+     path_to_app_gradle_file = File.expand_path(Dir.glob("../build.*").first, Dir.pwd)
+
+     new_app_version = values[:sdk_version]
+
+    build_sample_app(
+        sample_app_name: 'java_layout',
+        app_version: new_app_version,
+        firebase_test_groups: 'public',
```

Comment:
Can we publish to two groups here? It would be good to publish to 'Stable builds' in addition to 'Public' until we get all SDK's moved over to the 'public' group. Thoughts? If the effort is much more than adding another group to an array, let's chat as it might not be worth it.

Reply:
Sure thing, added it!